doc/dev/fpga-programming.md

```diff
@@ -100,6 +100,23 @@ Add user to plugdev group:
 sudo usermod -a $USER -G plugdev
 ```
 
+### Installing openFPGALoader (Linux/MacOS)
+
+The easiest way to install openFPGALoader is to use the Nix "develop" shell environment for this project.
+
+Install `nix`:
+```sh
+curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix | sh -s -- install
+
+# then restart shell or call
+. /nix/var/nix/profiles/default/etc/profile.d/nix-daemon.sh
+```
+
+Enter Nix shell from inside the sonata-system repository:
+```sh
+nix develop
+```
+
```

Comment:
Not sure if it makes sense to put Nix install instructions here. There is already mentions on how to get openFPGALoader in two places:
https://github.com/lowRISC/sonata-system/blob/4b72d8c07c727846c6ccb27754352388f3b2ac9a/doc/dev/fpga-development.md?plain=1#L9
and
https://github.com/lowRISC/sonata-system/blob/4b72d8c07c727846c6ccb27754352388f3b2ac9a/doc/guide/fpga-update.md?plain=1#L94